cypress/integration/buy-shirt.spec.ts

```diff
@@ -18,13 +18,30 @@ const paymentStepPage = new PaymentStepPage();
 
 describe("Buy a t-shirt", () => {
 
+    /* let menuContentPage: MenuContentPage;
+     let productListPage: ProductListPage;
+     let loginPage: LoginPage;
+     let shippingStepPage: ShippingStepPage;
+     let addressStepPage: AddressStepPage;
+     let paymentStepPage: PaymentStepPage;
+
+     before(() => {
+         menuContentPage = new MenuContentPage;
+         productListPage = new ProductListPage;
+         loginPage = new LoginPage;
+         shippingStepPage = new ShippingStepPage;
+         addressStepPage = new AddressStepPage;
+         paymentStepPage = new PaymentStepPage;
+     })*/
```

Comment:
Uncomment these lines and delete lines 10 to 16

Reply:
Done